docs/sources/reference/components/prometheus/prometheus.exporter.snmp.md

```diff
@@ -17,25 +17,34 @@ The `prometheus.exporter.snmp` component embeds
 
 ## Usage
 
+### Recommended usage
+
```

Comment:
I'd like to remove this and keep it just Usage to keep it consistent with the other topic layouts.